image.go

```diff
@@ -1012,18 +1012,20 @@ func imageSeenOrWait(ctx context.Context, opt *imageOpt, tag string, dig digest.
 //   - blobs/$algo/$hash: each content addressable object (manifest, config, or layer), created recursively
 //
 // [OCI Layout]: https://github.com/opencontainers/image-spec/blob/master/image-layout.md
-func (rc *RegClient) ImageExport(ctx context.Context, r ref.Ref, outStream io.Writer, opts ...ImageOpts) error {
-	if !r.IsSet() {
-		return fmt.Errorf("ref is not set: %s%.0w", r.CommonName(), errs.ErrInvalidReference)
+func (rc *RegClient) ImageExport(ctx context.Context, refs []ref.Ref, outStream io.Writer, opts ...ImageOpts) error {
```

Comment:
This is a breaking change that would impact anyone that has imported the library.

Reply:
I can add another function and implement this interface using that, if this is needed for other applications.